firecrown/likelihood/binned_cluster_number_counts.py

```diff
@@ -107,11 +107,11 @@ def _compute_theory_vector(self, tools: ModelingTools) -> TheoryVector:
             if cl_property == ClusterProperty.COUNTS:
                 theory_vector_list += cluster_counts
                 continue
-
+            if cl_property == ClusterProperty.DELTASIGMA:
```

Comment:
Before, this was giving an error if the property was neither COUNTS or MEAN_MASS. However, the way the code is structure now, I want to be able to pass two statistics for the likelihood, and this statistics will not do anything with the DELTASIGMA data type. It has not to give an error because the other statistics will read this data (I have one SACC file with all the three data types). Nonetheless, should we test it here if all the properties provided are at least in the ClusterProperty module? Is there a good way to do it besides adding a for loop here? @m-aguena @vitenti